django/db/models/options.py

```diff
@@ -147,20 +143,6 @@ def app_config(self):
     def installed(self):
         return self.app_config is not None
 
-    @property
-    def abstract_managers(self):
```

Comment:
After this patch we no longer differentiate between abstract or concrete managers. Luckily these weren't documented, so I think we can remove them.